pkg/customflag/validate.go

```diff
@@ -36,7 +36,11 @@ func ValidateTemplateFlags() {
 	case "components":
 		validateComponentsTest(expectedValues, expectedUpgrades)
 	case "flannel":
```

Comment:
we need to update here as well with the new tags 
https://github.com/rancher/distros-test-framework/blob/7eb69146098fe68ee67a80a1f51522dbd29391a0/pkg/customflag/validatejenkins.go#L80

Reply:
Updated